etna/models/holt_winters.py

```diff
@@ -609,18 +616,47 @@ def __init__(
             )
         )
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid doesn't tune ``seasonal_periods`` parameter. It expected to be set by the user.
 
-class HoltModel(HoltWintersModel):
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        if self.seasonal is None:
+            return {
+                "trend": CategoricalDistribution(["add", "mul", None]),
+                "damped_trend": CategoricalDistribution([False, True]),
+                "use_boxcox": CategoricalDistribution([False, True]),
+            }
+        else:
+            return {
+                "trend": CategoricalDistribution(["add", "mul", None]),
+                "damped_trend": CategoricalDistribution([False, True]),
+                "seasonal": CategoricalDistribution(["add", "mul", None]),
+                "use_boxcox": CategoricalDistribution([False, True]),
+            }
```

Comment:
Can we simplify this a bit? Just add `seasonal` to grid if `self.seasonal is not None`.
Also it looks like that `trend=None, damped_trend=True` is possible model evaluation configuration. In this case it will throw error.